Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -15,29 +15,46 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
 
     @IBOutlet weak var subheader: UILabel!
     @IBOutlet weak var lblWalletAddress: UILabel!
+    @IBOutlet var contentStackView: UIStackView!
     
     private var barButton: UIBarButtonItem?
-    
+    private lazy var syncInProgressLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .darkGray
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.text = "Please wait for your wallet to finish synchronizing."
+        return label
+    }()
+
     var firstTrial = true
     var starttime: Int64 = 0
     var myacc: WalletAccount!
     var account: WalletAccounts?
     var tapGesture = UITapGestureRecognizer()
     var oldAddress = ""
     var wallet = AppDelegate.walletLoader.wallet
-    
+    let isSynced = AppDelegate.walletLoader.isSynced
+    let isNewWalletSetup: Bool = Settings.readValue(for: Settings.Keys.NewWalletSetUp)
+
     private var selectedAccount = ""
     
     override func viewDidLoad() {
         super.viewDidLoad()
         self.subheader.text = "Each time you request a payment, a new address is created to protect your privacy."
         // TAP Gesture
         self.setupExtraUI()
-               self.showFirstWalletAddressAndQRCode()
-        self.populateWalletDropdownMenu()
         self.starttime = Int64(NSDate().timeIntervalSince1970)
+        self.toggleView()
+        setUpConstraints()
```

Comment:
Should probably
1) make the label hidden by default
2) setup constraints before "toggling" the view (i.e. before attempting to display the label)

In line with change 2 above, the `setUpConstraints` should be moved to above (before) the `toggleView` function